tests/typechecker/call_instance_method_statically.jakt

```diff
@@ -0,0 +1,16 @@
+/// Expect:
+/// - error: "Cannot call an instance method statically\n"
```

Comment:
Can we add some help text too?

Something like "did you mean to type `.b()`?"

Reply:
Currently I added the hint "Add a dot before the method name to call an instance method"